pkg/policy/background_test.go

```diff
@@ -0,0 +1,137 @@
+package policy
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	kyverno "github.com/kyverno/kyverno/pkg/api/kyverno/v1"
+	"gotest.tools/assert"
+)
+
+func Test_Validation_valid_backgroundPolicy(t *testing.T) {
```

Comment:
Can we add tests to have allowed tests in `rules.context.configmap`, `rules.context.apicall`, `rules.mutate`, `rules.validate`?